community-membership.md

```diff
@@ -1,77 +1,199 @@
-  mentor contributors and reviewers; demonstrated sound technical judgement; nominated
-  by an area/component owner, with no objections from other owners;  may be asked to
-  become an approver as a precondition for accepting a large code contribution
-- **Area/Component Owner**: in top-level [OWNERS](contributors/devel/owners.md) file for
-  some area/component as an approver; design/proposal approval authority for some area 
-  of the project, though escalation is still possible; expected to mentor and guide approvers,
-  reviewers, and other contributors; may be asked to become an area/component owner as a precondition
-  for accepting the contribution of a new component or other major function
-- [**kubernetes-maintainers**](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers):
-  approver for some part of the codebase for at least 3 months; on project for at least 1 year;
-  at least 50 merged and/or assigned PRs, including at least 20 as the primary reviewer;
-  write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki,
-  create/delete labels and milestones); technically can lgtm any PR and cause it
-  to be merged by the submit queue, but expected to respect OWNERS files; expected to review PRs, fix bugs, maintain and
-  improve health and quality of the project, provide user support, mentor and guide approvers,
-  reviewers, and other contributors; must apply to `kubernetes-maintainers@googlegroups.com`, with a
-  [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
-  kubernetes-maintainers members and a Sponsor from Project Approvers, with a summary
-  of contributions to the project, current project responsibilities, and links to merged and assigned PRs;
-  at least 3 of the maintainers must approve the application, with no objections; the application
-  expires after 2 weeks if not enough approvals are granted
-- **Project Approvers**: approver in [top-level OWNERS file in kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/OWNERS);
-  de-facto project decision makers; technically can 
-  approve virtually any PRs; can sponsor incubator repos; can sponsor maintainers;
-  maintainer in good standing for at least 1 year; strong technical vision;
-  committed to project's mission and culture; nomination/application process TBD
-- [**API Approver**](https://github.com/orgs/kubernetes/teams/api-approvers):
-  lead designers of the project, who are familiar with the 
-  design, requirements, mechanics, conventions, style, scope, gotchas, etc. 
-  of the API; most beta/GA API changes are vetted by the API approvers
-- [**API Reviewer**](https://github.com/orgs/kubernetes/teams/api-reviewers):
-  contributors familiar with design, requirements, mechanics, conventions, style,
-  scope, gotchas, etc. of the API; have written and/or reviewed Kubernetes APIs
\ No newline at end of file
+## Org member
```

Comment:
syntax nit: I think it's nice to have each step on the ladder be a single word.  Something we can get on shirts, etc.  This would unfortunately be "member". :)